tools/cov2json/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_swift_executable(cov2json
+  ProfileData.cpp
+  main.cpp
+  LINK_LIBRARIES swiftBasic
+  COMPONENT_DEPENDS support object profiledata "option" core)
+
+swift_install_in_component(compiler
```

Comment:
Do we need to ship this tool?  If not, please use another component, for example, `testsuite-tools`.